.golangci.yml

```diff
@@ -0,0 +1,63 @@
+run:
+  timeout: 5m
+
+linters:
+  disable-all: true
+  enable:
```

Comment:
Can you add a `TODO` below for GODRIVER-2156?

Reply:
Will add.